src/main/java/com/znsio/e2e/tools/JsonSchemaValidator.java

```diff
@@ -38,7 +38,7 @@ public static JSONObject validateJsonFileAgainstSchema(String jsonFilePath, Stri
     }
 
     private static InputStream loadJsonResourceFileAsStream(String fileName) {
-        String fileAsResource = File.separator + fileName;
+        String fileAsResource = "/" + fileName;
```

Comment:
String fileAsResource = File.separator + fileName;
is the right approach considering different platforms.

Reply:
There is apparently some issue on Windows using the above approach. Hence we had to revert that change